teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -80,13 +80,23 @@
       arity = "1")
   private int validatorExternalSignerTimeout = 1000;
 
+  @CommandLine.Option(
+      names = {"--Xvalidators-external-signer-concurrent-limit"},
+      paramLabel = "<INTEGER>",
+      description = "The maximum number of concurrent background requests to make to the signer.",
+      hidden = true,
```

Comment:
I seemed to hit timeouts fairly quickly, I wonder if that's a good reason to actually make this option not hidden?
Running everything on my machine with 64 validators, a queue of 64 was enough to start not completing within timeouts, and 50 was ok to complete signing without dropping anything, but this was on web3signer with no slashing protection enabled...

Reply:
I think the increased timeout will help.  I'd say keep it hidden to start with until we know more.  Wouldn't surprise me if we do make it a fully supported option at some point though.